Dp-202-shop/server/src/db/models/Product.model.ts

```diff
@@ -9,6 +9,10 @@ import {OrderItem} from "./OrderItem.model";
 
 export interface ProductAttributes {
   vendor_code: number;
+  // category_id? : number;
+  // manufacture_id?: number;
+  // user_id?: number;
+  // unit_id?: number;
   product_name: string;
```

Comment:
don't delete anything from db without informing. Put it back. It's the fields that are not necessary to enter